styleguide.md

```diff
@@ -18,4 +18,5 @@
 - Body: 
     Family: [Open Sans](https://fonts.google.com/specimen/Open+Sans)
     Size: 16px
-    Weight: 400 
\ No newline at end of file
+    Weight: 40
```

Comment:
Fix the typo font-weight. The value should be in the range of **100-900** at least